docs/releases/breaking-changes.md

```diff
@@ -13,6 +13,31 @@ these to a minimum, but rest assured we only make them with good reason.
 
 Starting with Directus 10.0, here is a list of potential breaking changes with remedial action you may need to take.
 
+## Vesion [NEXT]
+
+### Uninstalling registry extenstions via the API
+
+The logic to uninstall registry extensions has been moved from `/extensions/:uuid` to
+`/extensions/registry/uninstall/:uuid` and the old endpoint re-used to delete extensions from the database when the
+extension has gone missing from the filesystem.
+
```

Comment:
Do we actually consider this to be a breaking change? considering we have not yet documented the registry endpoints 🤔

Reply:
I don't think so 👍 We should update the docs soon, though...